packages/zoe/src/contracts/mintAndSellNFT.js

```diff
@@ -29,6 +29,7 @@ import { assert } from '@agoric/assert';
 const start = zcf => {
   const { tokenName = 'token' } = zcf.getTerms();
   // Create the internal token mint
+  // makeIssuerKit fails upgrade, safe because demo only?
```

Comment:
in this case yes

Reply:
marked contract as `notForProductionUse`